error-503.html

```diff
@@ -7,125 +7,159 @@
+        </ul>
+        <ul class="main-nav__account-menu account-menu">
+          <li><a href="https://www.nhs.uk/Personalisation/Login.aspx">Log in</a></li>
+          <li><a href="https://www.nhs.uk/Personalisation/Registration.aspx">Create an account</a></li>
+        </ul>
       </div>
-    </header>
+    </nav>
+    <!-- endnoindex -->
+  </header>
+
+
+      <main id="content" role="main" class="util-nofocus">
+        <div class="breadcrumb">
+          <ol>
+            <li><a href="http://www.nhs.uk">Home</a></li>
+            <li><a href="https://www.nhs.uk/service-search">Services near you</a></li>
 
-    <main id="content" role="main">
-      <div class="main--inner">
+              <li class="breadcrumb__last">Find a pharmacy</li>
+
+          </ol>
+        </div>
+
+        <div class="page-section">
         <h1>Sorry, we are experiencing technical problems.</h1>
         <p class="error-message">Please try again later.</p>
       </div>
+
     </main>
 
-    <footer role="contentinfo">
-      <div class="global-footer">
-        <div class="global-footer--inner">
-          <h2 class="util-visuallyhidden">Support links</h2>
-          <ul class="link-list">
-            <li>
-              <a href="https://beta.nhs.uk/help/clinical-assurance">Clinical assurance</a>
-            </li>
-            <li>
-              <a href="https://beta.nhs.uk/help/cookies">Cookies</a>
-            </li>
-          </ul>
-        </div>
-      </div>
+    <footer role="contentinfo" class="footer">
```

Comment:
Is the role attribute needed here?
![screen shot 2017-12-04 at 15 49 21](https://user-images.githubusercontent.com/639706/33561661-2c3f364a-d90b-11e7-9c3f-63bd7bdd69fc.png)

Reply:
Removed